Dockerfile

```diff
@@ -102,5 +102,9 @@ RUN apt-get -y install \
 # Create working directory for mounting the RIOT sources
 RUN mkdir -p /data/riotbuild
 
+# Set a global git user and email address
+RUN git config --global user.email "riot@riot-os.org"
+RUN git config --global user.name "RIOT-Docker"
```

Comment:
If you join the two lines with an `&&` we reduce the number of extra file system layers.